filebeat/filebeat.reference.yml

```diff
@@ -507,6 +507,16 @@ filebeat.prospectors:
   # Maximum size of the message received over UDP
   #max_message_size: 10240
 
+#------------------------------ Tcp prospector --------------------------------
+# Experimental: Config options for the TCP input
+#- type: tcp
+  #enabled: false
+  # The host and port to receive the new event
+  #host: "localhost:8080"
```

Comment:
Did you test to have multiple syslog inputs configured?  I assume the following should work:

```
- type: tcp
   host: "localhost:8080"
- type: tcp
   host: "localhost:8081"
```

Reply:
I haven't tested it, but it should work if every output have their own port.

Reply:
If both port are free:

```
2018-02-07T11:47:40.997-0500    INFO    tcp/harvester.go:58     Started listening for incoming TCP connection on: localhost:8080`
2018-02-07T11:47:40.997-0500    INFO    input/input.go:87       Starting input of type: tcp; ID: 14733937925171989371
2018-02-07T11:47:40.997-0500    DEBUG   [processors]    processors/processor.go:49      Processors:
2018-02-07T11:47:40.997-0500    INFO    tcp/input.go:50 Starting TCP input
2018-02-07T11:47:40.997-0500    INFO    tcp/harvester.go:58     Started listening for incoming TCP connection on: localhost:8081
2018-02-07T11:47:40.998-0500    DEBUG   [input] log/config.go:178       recursive glob enabled
```

if one port is already taken:

```
2018-02-07T11:48:52.928-0500    ERROR   instance/beat.go:674    Exiting: Error in initing input: listen tcp 127.0.0.1:8080: bind: address already in use
Exiting: Error in initing input: listen tcp 127.0.0.1:8080: bind: address already in use
```
FB exits